src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example2.java

```diff
@@ -0,0 +1,19 @@
+/* // violation 'File does not end with a newline.'
+NewlineAtEndOfFile
+lineSeparator = LF
+fileExtensions = (default)all files
+
+
+*/
+
+/*xml
+<module name="Checker">
+  <module name="NewlineAtEndOfFile"/>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.newlineatendoffile;
+
+public class Example2 { // ⤶
+// ⤶
+}
```

Comment:
Please make a comment `// no ⤶ below` to point user where is missing invisible symbol is located.

Please do same for other files.